cpp/unreal_plugins/CoreUtils/Source/CoreUtils/Config.h

```diff
@@ -17,6 +17,7 @@ class COREUTILS_API Config
 public:
     static void initialize();
     static void terminate();
+    static bool isEnabled();
```

Comment:
Update: no need for this trivial get function, we can just expose the bool directly as a public variable.